src/test/java/uk/gov/di/helpers/TestClock.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.di.helpers;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+
+public class TestClock extends Clock {
```

Comment:
I might be wrong, but I think this class does the same thing as the clock you get from the built-in `Clock.fixed(...)`

Reply:
Clock.fixed is immutable, so I can't change the time half way through the test as far as I'm aware?